binder/postBuild

```diff
@@ -32,11 +33,18 @@ else
   SPECIFIER="<="
 fi
 
-pip install plotly "pyspark[sql,ml,mllib,pandas_on_spark]$SPECIFIER$VERSION"
+if [[ $VERSION > "3.4.0" ]]; then
```

Comment:
I think it should be greater than or equal to? 3.4.0 version won't run this branch